src/rpc/rawtransaction.cpp

```diff
@@ -245,10 +245,11 @@ static UniValue gettxoutproof(const JSONRPCRequest& request)
     for (unsigned int idx = 0; idx < txids.size(); idx++) {
         const UniValue& txid = txids[idx];
         uint256 hash(ParseHashV(txid, "txid"));
-        if (setTxids.count(hash))
+        if (setTxids.count(hash)) {
             throw JSONRPCError(RPC_INVALID_PARAMETER, std::string("Invalid parameter, duplicated txid: ")+txid.get_str());
```

Comment:
fa46847c02d7f2c37c892bc03584f1eda8c21b34
While this is a style fixup commit, could this also be fixed
```suggestion
            throw JSONRPCError(RPC_INVALID_PARAMETER, std::string("Invalid parameter, duplicated txid: ") + txid.get_str());
```
?

Reply:
thx, added whitespace